contrib/kafka/config/SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security
+
+kafka channel controller and dispatcher connect to brokers without
+encryption and authentication by default. Encryption using SSL can be
+turned.
+
+## Connect over SSL(TLS)
+
+1. Create secret from Cluster CA public key to verify the identity of the Kafka brokers.
+
+   ```yaml
+   kubectl create secret generic kafka-certificates  --from-file=ca.crt -n knative-eventing
```

Comment:
What is this cert for? It suggests that it's a CA cert, but I think this might be a client cert? Any documentation about the format or where else this needs to get loaded from?